integration-tests/snapshots/ktaranov/sqlserver-kit/report.yaml

```diff
@@ -1563,11 +1562,10 @@ issues:
   - "PowerShell/WOX_Permissions.ps1:62:14    	Lsvr            	U   	foreach ($SQLsvr in get-content \"C:\	emp"
```

Comment:
Aside: looks like bad escaping converting \T from \Temp into a tab

Reply:
Yes, the report generator seems to have an issue.

Reply:
@nschonni,

Thank you for pointing it out. I fix the report.